src/misc/lv_fs.c

```diff
@@ -493,17 +499,37 @@ const char * lv_fs_get_last(const char * path)
  *   STATIC FUNCTIONS
  **********************/
 
+/**
+ * Checks if the path starts with a drive prefix (i.e. "X:..." or "Z/...")
+ * @param path path string (E.g. S:/folder/file.txt)
+ * @return number of characters making up the driver prefix (0 means no drive letter in the path)
+ */
+static int lv_fs_get_driver_prefix_length(const char * path)
+{
+    if(path[0] == '\0') return 0;
+
+    if(path[0] < 'A' || 'Z' < path[0]) return 0;
+
+    if(path[1] == ':')
+        return 2;
+    else if(path[1] == '/' || path[1] == '\\')
+        /* This case was introduced to the codebase at a7084509b51e8c39d4018144b38b5b6a4b5242f8,
```

Comment:
No need reference commits but rather explain the use case:
```suggestion
       /*For backward compatibility support the special case when `/` or `\` is used as driver letter. */
```

Shouldn't is check `path[0]`: